recipes/r-nimble/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set version = '0.9.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-nimble
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/nimble_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/nimble/nimble_{{ version }}.tar.gz
+  sha256: ad5e8a171193cb0172e68bf61c4f94432c45c131a150101ad1c5c7318c335757
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-r6
+    - r-coda
+    - r-igraph
+    - libopenblas
+    - liblapack
+    - libblas
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-r6
+    - r-coda
+    - r-igraph
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
```

Comment:
Are really all of those `build` dependencies also required here for `run`?

Reply:
@dbast Nimble converts R code to c++ and compiles it as part of its function, using `R CMD SHLIB` for linking, so it needs most of the tooling required to build an R source package.  It's possible that some of them are not necessary (`zip`), but the test failed on Windows with a smaller subset, so I tossed them all in rather than test each one individually (I don't have access to a local Windows machine.)

Reply:
@dbast I've now removed as many requirements as possible without breaking anything.